.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,90 @@
+<!-- Type of change
+Please label this PR with one of the following labels, depending on the scope of your change:
+- bug
+- enhancement
+- breaking change
+-->
+
+## What does this PR do?
+
+<!-- Mandatory
+Explain here the changes you made on the PR. Please explain the WHAT: patterns used, algorithms implemented, design architecture, message processing, etc.
+
+Example:
+  Expose 'xpack.monitoring.elasticsearch.proxy' in the docker environment variables and update logstash.yml to surface this config option.
+  
+  This commit exposes the 'xpack.monitoring.elasticsearch.proxy' variable in the docker by adding it in env2yaml.go, which translates from
+  being an environment variable to a proper yaml config.
+  
+  Additionally, this PR exposes this setting for both xpack monitoring & management to the logstash.yml file.
+-->
+
+## Why is it important/What is the impact to the user?
+
+<!-- Mandatory
+Explain here the WHY or the IMPACT to the user, or the rationale/motivation for the changes.
+
+Example:
+  This PR fixes an issue that was preventing the docker image from using the proxy setting when sending xpack monitoring information.
+  and/or
+  This PR now allows the user to define the xpack monitoring proxy setting in the docker container.
+-->
+
+## Checklist
+
+<!-- Mandatory
+Add a checklist of things that are required to be reviewed in order to have the PR approved
+
+List here all the items you have verified BEFORE sending this PR. Please DO NOT remove any item, striking through those that do not apply. (Just in case, strikethrough uses two tildes. ~~Scratch this.~~)
+-->
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
```

Comment:
I do a lot of the bigger docs changes outside of the code PRs (monitoring, API keys, etc.) Maybe add, "or consulted with the tech writer on docs changes?"

Reply:
I fill like this should be implicit. I mean, for them to have made the changes, they should have either reviewed this with you, or are asking you to review as part of this branch.